.github/workflows/build.yml

```diff
@@ -20,6 +20,10 @@ jobs:
         echo " "
         which pylint
 
+        echo " "
+        ls -aFlrt
```

Comment:
Just to check the contents of the repo, now that we are creating LineOfCode as a sub-module of this repo.